Sources/Rendering/Core/Actor/index.js

```diff
@@ -102,18 +102,18 @@ function vtkActor(publicAPI, model) {
       vtkDebugMacro('Recomputing bounds...');
       model.mapperBounds = bds.concat(); // copy the mapper's bounds
       const bbox = [
-        vec3.fromValues(bds[1], bds[3], bds[5]),
-        vec3.fromValues(bds[1], bds[2], bds[5]),
-        vec3.fromValues(bds[0], bds[2], bds[5]),
-        vec3.fromValues(bds[0], bds[3], bds[5]),
-        vec3.fromValues(bds[1], bds[3], bds[4]),
-        vec3.fromValues(bds[1], bds[2], bds[4]),
-        vec3.fromValues(bds[0], bds[2], bds[4]),
-        vec3.fromValues(bds[0], bds[3], bds[4]),
+        new Float64Array([bds[1], bds[3], bds[5]]),
+        new Float64Array([bds[1], bds[2], bds[5]]),
+        new Float64Array([bds[0], bds[2], bds[5]]),
+        new Float64Array([bds[0], bds[3], bds[5]]),
+        new Float64Array([bds[1], bds[3], bds[4]]),
+        new Float64Array([bds[1], bds[2], bds[4]]),
+        new Float64Array([bds[0], bds[2], bds[4]]),
+        new Float64Array([bds[0], bds[3], bds[4]]),
```

Comment:
Could be written like 

```
const bbox = [
   vec3.set(new Float64Array(3), bds[1], bds[3], bds[5]),
   vec3.set(new Float64Array(3), bds[1], bds[2], bds[5]),
   ...
]
```

Reply:
But I'm assuming this is not called in a tight loop, so it doesn't matter. Otherwise you should probably re-use those Float64Arrays...

Reply:
It should call BoundingBox.getCorners probably as that is what it is doing and do it on one single Float64Array(24)